airflow_provider_fivetran/hooks/fivetran.py

```diff
@@ -1,23 +1,24 @@
+import json
+import logging
 from time import sleep
-from urllib.parse import urlparse
 
 import requests
 from requests import PreparedRequest, exceptions as requests_exceptions
 from requests.auth import AuthBase
 import pendulum
-import logging
-import json
-import time
-
 
-from airflow import __version__
+# from airflow import __version__
 from airflow.exceptions import AirflowException
-from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.base import BaseHook
 
 log = logging.getLogger(__name__)
```

Comment:
@pgzmnk Question for you, is this necessary? This gets setup in the BaseHook as well, and then there is the logger mixin that we are ultimately inheriting.

Reply:
Speaking about instantiating a logger in this file, `log = logging.getLogger(__name__)`